src/renderers/shared/utils/CallbackQueue.js

```diff
@@ -41,23 +43,23 @@ Object.assign(CallbackQueue.prototype, {
    * @param {?object} context Context to call `callback` with.
    * @internal
    */
-  enqueue: function(callback, context) {
+   enqueue(callback: () => void, context: T) {
     this._callbacks = this._callbacks || [];
-    this._contexts = this._contexts || [];
     this._callbacks.push(callback);
+    this._contexts = this._contexts || [];
```

Comment:
Yeah, flow doesn't know that push() isn't messing up with everything. I figured moving it one line after isn't a big deal to make flow happy